packages/sanity/src/core/form/inputs/PortableText/object/BlockObject.tsx

```diff
@@ -319,7 +318,7 @@ export const DefaultBlockObjectComponent = (props: BlockProps) => {
         {renderPreview({
           actions: (
             <BlockObjectActionsMenu
-              focused={focused}
+              blockFocused={focused}
```

Comment:
Renamed this prop to make it clearer that it refers to whether the surrounding _block_ is having focus (can revert if it has unintended consequences)